.github/workflows/ci.yml

```diff
@@ -176,38 +191,49 @@ jobs:
 
   build-docker-image:
     runs-on: ubuntu-22.04
+    outputs:
+      version: ${{ steps.version.outputs.VERSION_TAG }}
     needs:
       - backend-unit-tests
       - frontend-unit-tests
       - frontend-e2e-tests
       - build-skip-check
     if: needs.build-skip-check.outputs.skip == 'false'
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - linux/amd64
+          - linux/arm64
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         with:
           fetch-depth: 1
-      - uses: actions/setup-node@v3
+      - uses: actions/download-artifact@v3
         with:
-          node-version: ${{ env.NODE_VERSION }}
-          cache: 'yarn'
-      - name: Install Dependencies
-        run: |
-          npm install --global --force yarn@1.22.19
-          yarn cache clean && yarn --frozen-lockfile --network-concurrency 1
-
+          name: frontend
```

Comment:
I'm not 100% clear where this `frontend` gets used in the build. Would you mind explaining? Does this also get pushed to Dockerhub as part of the `preview` image?

Reply:
How do we build this locally, then? `docker build -t redash_test .` seems to not build the frontend